doc/source/whatsnew/v0.16.1.txt

```diff
@@ -52,6 +52,7 @@ Enhancements
 - ``Period`` now accepts ``datetime64`` as value input. (:issue:`9054`)
 
 - Allow timedelta string conversion when leading zero is missing from time definition, ie `0:00:00` vs `00:00:00`. (:issue:`9570`)
+- ``Series.corr()`` now supports `Biweight Midcorrelation`_, a median-based correlation metric, a robust alternative to Pearson's R.
```

Comment:
show the actual argument name, e.g. ``'biweight'`` (you can still point to the article)